### ISSUE TITLE
Hue v2 API support?

### ISSUE THREAD

**Issue by darrenparkinson** [amimof/huego#50]
- [X] This feature/enhancement benefits everyone

**Description**

Hi. Really love this library. I was wondering if you have any plans to support the v2 Hue API. I can’t see anything browsing from my mobile. Thanks for any feedback you can provide. 

**Additional information**

**Comment by amimof** [comment]
I just started working on this and it is on it’s way. Would you feel comfortable in contributing code? Would perhaps need some help 😊

**Comment by darrenparkinson** [comment]
Yes, of course, if there’s something I can help with, I’d love to.

**Comment by amimof** [comment]
@darrenparkinson A WIP PR is now up: #52 
You are more than welcome to try it out as it's beeing developed. Would appreciate input.
